docs/src/architecture/data-model.md

```diff
@@ -0,0 +1,253 @@
+
+# thin-edge data-model
+
+The **thin-edge data-model** is used to represent all device-related information.
+It covers the thin-edge device itself, as well as other _external devices_ connected to thin-edge.
+Each device is represented with a data **object**. 
+Each data **object** can have different **fields**. 
+The following JSON code shows a small example of a device data **object**:
+       
+```json
+          {
+              "name": "thin-edge device",
+              "type": "thin-edge.io"
+          }
+```
+
+**thin-edge** manages an **inventory** to store and provide all those objects. 
+
+The figure below illustrats the **data-model** objects and the **inventory**:
```

Comment:
For me this is more a deescription of "domain-model" rather than "data-model". That also begs the question elements of the data model e.g. measurements, alarms etc are they also part of the inventory?